vocabularies/trs.ttl

```diff
@@ -67,8 +67,9 @@ trs:BeforePresent
   a skos:Concept ;
   a time:TRS ;
   rdfs:seeAlso <https://en.wikipedia.org/wiki/Before_Present> ;
+  skos:broader trs:BeforePresent ;
```

Comment:
remove skos:broader trs:BeforePresent so it isn't a circular broader/narrower relationship.

Reply:
In geological and archeological circles, "Before Present" is a specific type of timescale, not a category of timescales. So there needs to be a "Before Present" timescale under the "Before Present" category. I'm not sure how this should be done. Can you change the category "Before Present" to  "Geologic and Archeologic Timescales" or would that be too complicated. We could also call the "Before Present" timescale "Before Present Uncalibrated"

Reply:
Categories are best represented by creating a Skos Collection.
for example 
- BeforePresent is a concept of a specific timescale
- BeforePresentCalibrated is a narrower more tightly defined type of concept within the broader BeforePresent scale (i.e. it is before present, but with the additional requirement of calibration)
- Both would be members of the categorical collection "Geologic and Archeologic Timescales"

Reply:
To adhere to skos formats but retain the intent of you addition:
delete skos:broader trs:BeforePresent

but at the bottom of the file create a collection of "Geologic and Archeologic Timescales" e.g. 

trs:geo-arch-trs a skos:Collection
  skos:definition "Timescales used in Geologic and Archeologic disciplines"@en ;
  skos:member trs:BeforePresent ,
    trs:BeforePresentCalibrated ,
    trs:GeologicTimescale ,
    trs:ThousandsOfYearsAgo ,
    trs:MillionsOfYearsAgo ,
    trs:BillionsOfYearsAgo ;
  skos:prefLabel "Geologic and Archeologic Temporal References"@en;
.
.